test/unit/store/item-store.js

```diff
@@ -75,4 +75,25 @@ describe('ItemStore', () => {
         });
     });
 
+    it('should delete an item', () => {
+        assert.strictEqual(items.length, 3);
+        return addAction('DELETE_ITEM', {
+            itemAddress: items[0].address
+        }).then(() => {
+            items = ItemStore.getItems();
+            assert.strictEqual(items.length, 2);
```

Comment:
I argue that you have a test asserting an item with the deleted address is no longer in `ItemStore.getItems()`, so that somebody doesn't accidentally misunderstand our API and delete the wrong item.